repo2docker/docker.py

```diff
@@ -69,6 +74,14 @@ class DockerEngine(ContainerEngine):
         config=True,
     )
 
+    extra_buildx_build_args = List(
```

Comment:
Is it worth pushing this up a level to `ContainerEngine` under a more generic name? It shouldn't be a breaking change since it'll be ignored by default.